nsqd/protocol_v2.go

```diff
@@ -399,7 +399,7 @@ func (p *protocolV2) IDENTIFY(client *clientV2, params [][]byte) ([]byte, error)
 		if identifyData.DeflateLevel <= 0 {
 			deflateLevel = 6
```

Comment:
Instead of that, we should drop the change on line 402 and update line 398 to:

```go
deflateLevel := identifyData.DeflateLevel
```

Otherwise the JSON response below would be broken.